src/components/middle/message/Reactions.tsx

```diff
@@ -49,6 +51,7 @@ const Reactions: FC<OwnProps> = ({
           withRecentReactors={totalCount <= MAX_RECENT_AVATARS}
           genericEffects={genericEffects}
           observeIntersection={observeIntersection}
+          hideAvatars={hideAvatars}
```

Comment:
It looks like `withRecentReactions` already does the same.

Reply:
done